app/assets/javascripts/admin/addon/components/admin-badges.gjs

```diff
@@ -75,6 +83,7 @@ export default class AdminBadges extends Component {
           <NavItem
             @route="adminBadges.index"
             @label="admin.config.badges.title"
+            @currentWhen={{this.activeBadges}}
```

Comment:
I wonder if it would be better to make this work the [same way as LinkTo](https://api.emberjs.com/ember/release/classes/Ember.Templates.components/methods/input?anchor=LinkTo#keepingalinkactiveforotherroutes)? So you can either supply a boolean, or a space-separated list of route names?